OffchainLabs/arbitrum-token-bridge
# fix: custom orbit chain edge case

### Summary

When users add a custom chain pair and make a deposit, it will be cached. If they then remove the custom orbit chain, transaction history will throw errors because we are trying to access a chain that does not exist.

Also updated the row key because react-virtualized key is not reliable here. Causes some side effects.



### Steps to test

1. Add a custom Orbit chain.
2. Make a deposit (you don't have to wait for it to complete).
3. Remove the custom Orbit chain.
4. Transaction history should now throw. (after the fix it shouldn't throw and the tx history shouldn't include those txs)
